plugins/texteditor.koplugin/main.lua

```diff
@@ -33,7 +33,7 @@ local TextEditor = WidgetContainer:new{
 }
 
 function TextEditor:onDispatcherRegisterActions()
-    Dispatcher:registerAction("edit_last_edited_file", { category = "none", event = "OpenLastEditedFile", title = _("Texteditor: open last file"), device = true, separator = true, })
+    Dispatcher:registerAction("edit_last_edited_file", { category = "none", event = "OpenLastEditedFile", title = _("Text editor: open last file"), device = true, separator = true, })
```

Comment:
II would use the coding style of `dispatcher.lua` here, so `{category = ` ... `separator = true})`

Reply:
Done